spec/lib/license_finder/package_managers/composer_spec.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'fakefs/spec_helpers'
+require 'json'
+
+module LicenseFinder
+  describe Composer do
+    let(:root) { '/fake-composer-project' }
+    let(:composer) { Composer.new project_path: Pathname.new(root) }
+
+    it_behaves_like 'a PackageManager'
+
+    let(:composer_shell_command_output) do
+      {
+        'require' => {
+          'vlucas/phpdotenv' => '3.3.*'
+        },
+        'require-dev' => {
+          'symfony/debug' => '4.2.*'
+        }
+      }.to_json
+    end
+
+    describe '.prepare' do
```

Comment:
Looks like the current_packages test was removed? Why was this? It probably should be added in addition to the prepare test.

Reply:
Looks like I overlooked that because tests were failing so bad, I will see how hard it is to put current_packages check back in.

Reply:
I've added back in the tests and fixed a couple issues that popped up from it, should be good to go once the pipeline clears.